addons/test_discuss_full/tests/test_performance.py

```diff
@@ -306,6 +306,8 @@ def _add_reactions(self, message, reactions):
                     "action": "add",
                     "content": reaction,
                     "message_id": message.id,
+                    "thread_id": message.res_id,
+                    "thread_model": message.model,
```

Comment:
as posted in JS, these params are not necessary as they directly depend on message